### PR TITLE
7alip | Week1

### PR DESCRIPTION
**Pull request by 7alip**

### CODE REVIEW

File: week1/homework/src/render.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const render = (response, state, error = false) => {
+  response.setHeader('Content-Type', 'application/json');
```
Comment (gajduk) [inline week1/homework/src/render.js]: I love that you have separated this code from the rest and reused it 🥇

File: week1/homework/src/server.js [hunk 2]
```diff
@@ -9,7 +10,26 @@ function createServer(port) {
   let state = 10;
 
   const server = http.createServer((request, response) => {
-    // TODO: Write your homework code here
+    switch (request.url) {
+      case '/state':
+        render(response, state);
```
Comment (gajduk) [inline week1/homework/src/server.js]: very tidy 👍

File: week1/homework/src/server.js [hunk 3]
```diff
@@ -9,7 +10,26 @@ function createServer(port) {
...
+        break;
+      case '/subtract':
+        render(response, --state);
+        break;
+      case '/reset':
+        state = 10;
+        render(response, state);
+        break;
+      default:
+        response.statusCode = 404;
```
Comment (gajduk) [inline week1/homework/src/server.js]: 404 👍

**Comment by 7alip** [comment]
Hi @gajduk. Thank you for your nice comments.

**Pull request closed without merging**